conan/tools/cmake/toolchain/blocks.py

```diff
@@ -682,7 +682,7 @@ def context(self):
         compilers = {}
         # Allowed <LANG> variables (and <LANG>_LAUNCHER)
         compilers_mapping = {"c": "C", "cuda": "CUDA", "cpp": "CXX", "objc": "OBJC",
-                             "objcxx": "OBJCXX", "rc": "RC", 'fortran': "Fortran", 'asm': "ASM",
+                             "objcpp": "OBJCXX", "rc": "RC", 'fortran': "Fortran", 'asm': "ASM",
```

Comment:
it was weird to have `cpp` and `objcxx`

Reply:
Maybe we should have a small test that covers that the mapping is good ?